AutomatedTesting/Gem/PythonTests/Atom/atom_utils/material_editor_utils.py

```diff
@@ -3,281 +3,29 @@
-            and is_close(color1.g, color2.g, buffer)
-            and is_close(color1.b, color2.b, buffer)
-    )
-
-
-def open_material(file_path):
-    """
-    :return: uuid of material document opened
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "OpenDocument", file_path)
-
-
-def is_document_open(document_id):
-    """
-    :return: bool
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "IsDocumentOpen", document_id)
-
-
-def save_document(document_id):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "SaveDocument", document_id)
-
-
-def save_document_as_copy(document_id, target_path):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(
-        bus.Broadcast, "SaveDocumentAsCopy", document_id, target_path
-    )
-
-
-def save_document_as_child(document_id, target_path):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(
-        bus.Broadcast, "SaveDocumentAsChild", document_id, target_path
-    )
-
-
-def save_all():
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "SaveAllDocuments")
-
-
-def close_document(document_id):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "CloseDocument", document_id)
-
-
-def close_all_documents():
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "CloseAllDocuments")
-
-
-def close_all_except_selected(document_id):
-    """
-    :return: bool success
-    """
-    return azlmbr.atomtools.AtomToolsDocumentSystemRequestBus(bus.Broadcast, "CloseAllDocumentsExcept", document_id)
-
-
-def get_property(document_id, property_name):
+def get_property(document_id: azlmbr.math.Uuid, property_name: str) -> any:
     """
+    Gets a property value for a given document_id and property name.
+    :param document_id: The UUID of a given document file.
+    :param property_name: The property name to search for the property value.
     :return: property value or invalid value if the document is not open or the property_name can't be found
     """
-    return azlmbr.materialeditor.MaterialDocumentRequestBus(bus.Event, "GetPropertyValue", document_id, property_name)
+    return materialeditor.MaterialDocumentRequestBus(bus.Event, "GetPropertyValue", document_id, property_name)
 
 
-def set_property(document_id, property_name, value):
-    azlmbr.materialeditor.MaterialDocumentRequestBus(bus.Event, "SetPropertyValue", document_id, property_name, value)
-
-
-def is_pane_visible(pane_name):
+def set_property(
+        document_id: azlmbr.math.Uuid, property_name: str, value: math.Uuid or int or float or math.Color) -> None:
```

Comment:
get and set property both work with any, bool, int, uint, float, color, vector 2, vector 3, vector 4, string, or asset ID